include/lo2s/summary.hpp

```diff
@@ -0,0 +1,26 @@
+#include <boost/dynamic_bitset.hpp>
+#include <ctime>
+namespace lo2s
+{
+class Summary
+{
+public:
+    static void start();
+    static void finalize_and_print();
+    static void record_wakeups(int num_wakeups);
+    static void increase_thread_count();
+    static void set_trace_dir(std::string trace_dir);
+    static void set_pid(pid_t pid);
```

Comment:
I'd rename it to a more high-level descriptive name like `register_process` or something.